_stbt/core.py

```diff
@@ -533,14 +535,51 @@ def _load_template(template):
     if isinstance(template, numpy.ndarray):
         return _AnnotatedTemplate(template, None, None)
     else:
-        template_name = _find_path(template)
-        if not template_name or not os.path.isfile(template_name):
-            raise UITestError("No such template file: %s" % template)
-        image = cv2.imread(template_name, cv2.CV_LOAD_IMAGE_COLOR)
+        relative_filename = template
+        absolute_filename = _find_user_file(relative_filename)
+        if not absolute_filename:
+            raise IOError("No such template file: %s" % relative_filename)
```

Comment:
This should be `IOError(errno.ENOENT, "No such template file: %s" % relative_filename, relative_filename)`

[Python docs say](https://docs.python.org/2/library/exceptions.html#exceptions.EnvironmentError):

> When exceptions of this type are created [...], the first item is available on the instance’s errno attribute (it is assumed to be an error number), and the second item is available on the strerror attribute \[...\]  the third item is available on the filename attribute.